modules/nw-rfhe-creating-hardware-event-kind-cr.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
+:_content-type: PROCEDURE
+[id="nw-rfhe-hardware-event-kind-cr_{context}"]
+= Creating the HardwareEvent resource
+
+To start using Baremetal Hardware Event proxy, install the operator and then create the following HardwareEvent.
+
+
+[NOTE]
+====
+You can only create one instance of HardwareEvent in this release.
+====
+
+[source,yaml]
+----
+    apiVersion: "event.redhat-cne.org/v1alpha1"
+    kind: "HardwareEvent"
+    metadata:
+      name: "hardware-event"
+    spec:
+     nodeSelector: {}
```

Comment:
Can't we include the `nodeSelector` field here and add a note that you select the node to register Redfish events by applying the label? Then we can remove the same CR below.
```
  nodeSelector:
    node-role.kubernetes.io/hw-event: ""
```